src/components/resource/form.js

```diff
@@ -161,6 +163,18 @@ defineComponent('furet-ui-resource-form', {
           })
         }
       },
+      refresh_fields () {
+        this.$emit('clear-change', {pks: this.pks, uuid: this.uuid})
+        this.readonly = true;
+        console.log(this.fields2refresh)
+        this.fields2refresh.forEach(obj => {
+          obj.refresh()
```

Comment:
I'm not sure if the scope would be properly preserved if we registered a callback method instead the entire object. As far a I know there is no way to force user to implement refresh method if it register the object.

or at leat add a nice message to developer to tell them to implement refresh method if `registryField2Refresh` is used